### PR TITLE
Feat/react router

### PR DESCRIPTION
**Pull request by hellodebs**
Hey there, here's is my PR. Please review :)

- fetched an listed characters from the Ricky and Morty API
- made each character from whole list clickable so it renders an individual profile of each character

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+        <nav className="app__nav">
+          <NavLink className="app__nav-link" to="/">
+            Home
+          </NavLink>
+          <NavLink className="app__nav-link" to="/AllCharacters">
+            All characters
+          </NavLink>
+        </nav>
+      </header>
+      <main className="app__main">
```
Comment (mikemangas) [inline src/App.js]: nice bem naming

File: src/components/CharacterCard.js [hunk 2]
```diff
@@ -2,9 +2,9 @@ import "./CharacterCard.css";
...
   return (
-    <main className="app__character-card">
-      <div>CharacterCard</div> <div>CharacterCard</div> <div>CharacterCard</div>
-      <div>CharacterCard</div>
-    </main>
+    <div className="app__character-card">
+      <p>CharacterCard</p> <p>CharacterCard</p> <p>CharacterCard</p>
+      <p>CharacterCard</p>
+    </div>
   );
```
Comment (mikemangas) [inline src/components/CharacterCard.js]: why do you have so many character card p tags? do you just test stuff?

Comment (hellodebs) [reply src/components/CharacterCard.js]: oh, somehow you commented on my outdated code. Please check `files changed` :) But yes, I used it for my static version.

**Review by ScienceMaster7 — APPROVED** [review]
looks good

**Pull request merged**